interactive_ai/services/media/Makefile

```diff
@@ -3,5 +3,6 @@
 
 include ../../../Makefile.shared-go
 
+COMPONENT_NAME := media-ms
```

Comment:
The other services don't have the `-ms` suffix. How much effort would it be to remove it?

Reply:
Thanks, Leo! I will create a ticket to unify all component names as I also noticed some components are named in a snake case notation like `weights_downloader`. Regarding efforts - we'll have to configure the container registry.